### PR TITLE
HOTFIX ticket export xlsx format test

### PR DESCRIPTION
**Pull request by pahaz**

### CODE REVIEW

File: apps/condo/domains/ticket/schema/TicketExportTask.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const { normalizeTimeZone } = require('@condo/domains/common/utils/timezone')
 const { DEFAULT_ORGANIZATION_TIMEZONE } = require('@condo/domains/organization/constants/common')
 const { extractReqLocale } = require('@condo/locales/extractReqLocale')
 const conf = require('@condo/config')
-const { exportTicketsTask } = require('../tasks/exportTicketsTask')
+const { exportTickets } = require('../tasks')
```
Comment (sitozzz) [inline apps/condo/domains/ticket/schema/TicketExportTask.js:17]: Absolute import)

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/TicketExportTask.test.js [hunk 2]
```diff
@@ -241,3 +251,118 @@ describe('TicketExportTask', () => {
     })
 
 })
+
+const downloadFile = async (url, path) => {
+    const res = await fetch(url)
+    if (res.status !== 200) throw new Error(`downloadFile(): status=${res.status}`)
+    const buffer = await res.buffer()
+    fs.writeFileSync(path, buffer)
+}
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/TicketExportTask.test.js:260]: Maybe to extract it to `common` domain into utils?

File: apps/condo/domains/ticket/schema/TicketExportTask.test.js [hunk 3]
```diff
@@ -241,3 +251,118 @@ describe('TicketExportTask', () => {
...
+
+const readXlsx = async (path) => {
+    const workbook = xlsx.readFile(path)
+    const sheet = workbook.Sheets[workbook.SheetNames[0]]
+    return xlsx.utils.sheet_to_json(sheet, { header: 1, defval: '' }).filter(x => x.join(''))
+}
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/TicketExportTask.test.js:266]: Maybe to extract it to `common` domain into utils?

File: apps/condo/domains/ticket/schema/TicketExportTask.test.js [hunk 4]
```diff
@@ -241,3 +251,118 @@ describe('TicketExportTask', () => {
...
+
+const expectDataFormat = (data, res) => {
+    expect(JSON.stringify(data, null, 2)).toEqual(JSON.stringify(res, null, 2))
+}
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/TicketExportTask.test.js:270]: Maybe to extract it to `common` domain into utils?

**Review by AntonAL — APPROVED** [review]

**Pull request merged**